.github/workflows/unit-test.yml

```diff
@@ -90,19 +98,25 @@ jobs:
           ref: ${{ inputs.branch }}
           fetch-depth: ${{ inputs.checkout_fetch_depth }}
           repository: PennyLaneAI/pennylane
-
+      
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: ${{ inputs.uv_version }}
+          enable-cache: true
+          cache-dependency-glob: 'requirements**.txt'
```

Comment:
Do we want all the req files?

Reply:
I leave that call up to the team!

Reply:
I reckon the `requirements-ci.txt` should be enough here.

Reply:
Sounds good I can make that update

Reply:
Should I use requirements-*.txt in this case? The goal mainly being cache the libraries we list in any requirements so they are faster to install (when not changing)

Reply:
I'd say so

Reply:
Done!